deployment/kubernetes/yaml/cdcservice.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-cdc-service

```

Comment:
This application is called `eventuate-tram-examples-customers-and-orders` - not `ftgo` (name of the other application). However, that's a long name. I'd remove the `ftgo-` prefix for all k8s objects and define an `eventuate-tram-examples-customers-and-orders` namespace

Reply:
removed fgto- prefixes and deployed to `eventuate-tram-examples-customers-and-orders`namespace